src/main/java/models/Player.java

```diff
@@ -0,0 +1,59 @@
+package models;
+
+
+import java.util.ArrayList;
+
+/**
+ * Created by isaac on 3/12/16.
+ */
+public abstract class Player
+{
+    //"oops" not in the constructor
+    public java.util.List<Card> hand = new ArrayList<>();
+
+    public int score;
+
+    public void empty_hand()
+    {
+        int hand_size = hand.size();
+        for (int i = 0; i < hand_size; i++)
+            hand.remove(0);
+    }
+
+    public int setCardVal(Card card){
```

Comment:
Might one to add a test for this method later but other than that, looks good! +1